src/Misc/externals.sh

```diff
@@ -127,6 +127,10 @@ function acquireExternalTool() {
     fi
 }
 
+# Download the external tools for all platforms
+acquireExternalTool "$CONTAINER_URL/NuGet/3.5.1-beta1-1621/NuGet.zip" nuget
+acquireExternalTool "$CONTAINER_URL/CredentialProvider.TeamBuild/15.104.25518.0/CredentialProvider.TeamBuild.zip" nuget
```

Comment:
The credential provider itself does work under linux -- it's a super simple program. mono has a behavioral difference from regular .net which breaks actually retrieving packages from VSTS though, at least on whatever version is in Ubuntu 14.04.